scripts/zones/Aydeewa_Subterrane/mobs/Pandemonium_Warden.lua

```diff
@@ -32,9 +87,12 @@ end;
 
 function onMobEngaged(mob,target)
     -- pop pets
-    for i = 17056170, 17056177, 1 do
-        SpawnMob(i,180):updateEnmity(target);
-        GetMobByID(i):setModelId(1841);
+    for i = 1, 8 do
+        local pet = GetMobByID(petIDs[1][i]);
+        pet:setModelId(1841);
+        pet:spawn();
+        pet:updateClaim(target);
```

Comment:
Hmm... so there's also that block of code that was left over from previous version where if it's not time for a phase change or an astral flow it just checks to make sure the pets are engaged and if not it teleports them to PW and engages them on the tank. This is to prevent people from dragging the adds off and leaving them somewhere. If I spawn them unclaimed, they'll get hit by that code and engaged anyways (I think, at work, can't test). Should I remove that block of code too? They spawn on top of PW each phase regardless of where they are so they'll be able to engage normally. If they get drug off it'll just be for one phase which isn't a big deal I guess. The longest phase is only 20k hp.